daemon/cmd/daemon_main.go

```diff
@@ -770,7 +771,7 @@ func init() {
 	flags.MarkHidden(option.PolicyTriggerInterval)
 	option.BindEnv(option.PolicyTriggerInterval)
 
-	flags.Bool(option.DisableCNPStatusUpdates, false, `Do not send CNP NodeStatus updates to the Kubernetes api-server (recommended to run with "cnp-node-status-gc=false" in cilium-operator)`)
+	flags.Bool(option.DisableCNPStatusUpdates, false, fmt.Sprintf(`Do not send CNP NodeStatus updates to the Kubernetes api-server (recommended to run with "%s=0" in cilium-operator)`, operatorOption.CNPNodeStatusGCInterval))
```

Comment:
I'd rather keep the verbatim option here to avoid pulling in the `operator/option` dependency into the agent binary.

```suggestion
	flags.Bool(option.DisableCNPStatusUpdates, false, `Do not send CNP NodeStatus updates to the Kubernetes api-server (recommended to run with "cnp-node-status-gc-interval=0" in cilium-operator)`)
```